smtp/init.lua

```diff
@@ -155,6 +155,11 @@ curl_mt = {
             if not body or not url or not from then
                 error('request(url, from, to, body [, options]])')
             end
+            if (not to or not (to and to == {} or #to > 0)) and
+               (not opts.cc or not (opts.cc and opts.cc == {} or #opts.cc > 0)) and
+               (not opts.bcc or not (opts.bcc and opts.bcc == {} or #opts.cc > 0)) then
```

Comment:
```
tarantool> {} == {}
---
- false
...
```

The standard trick for checking a table for emptiness is `next(my_table) == nil`. However `#to > 0` should be enough in the case.